kedro-datasets/setup.py

```diff
@@ -184,7 +184,7 @@ def _collect_requirements(requires):
     "coverage[toml]",
     "dask[complete]>=2021.10",
     "delta-spark>=1.0, <3.0",
-    "deltalake>=0.10.0",
+    "deltalake>=0.10.0, <0.15.2",
```

Comment:
Doing one test here, one moment

```suggestion
    "deltalake>=0.10.0, !=0.15.2",
```